tests/ConfigTest.php

```diff
@@ -223,4 +223,17 @@ public function provideRegisterCustomFixersCases()
             [$fixers, new \ArrayIterator($fixers)],
         ];
     }
+
+    /**
+     * @group legacy
+     * @expectedDeprecation PhpCsFixer\Config::create is deprecated since 2.17 and will be removed in 3.0.
```

Comment:
almost, you've to move the comment to the test that uses `::create` as that is what you are deprecating, the constructor used here is the correct one :)

Reply:
(and can you update following this https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4828#discussion_r382065909 maybe? thanks!)

Reply:
Note that builds are failing because there are several other places in the code where the deprecated method is called.

Reply:
Yep, just saw that. Except I can't track where. Kinda puzzled here, had a short night ^^

Reply:
Oh *snap*, I got tricked by the `*.php` filter in PHPStorm. 
I really need to sleep, sometimes. ^^

Reply:
@julienfalque I'm kinda puzzled for the remaining calls here.

Says `testResolveIntersectionOfPaths()` invokes it, can't see even a generic call to it. Does the iterator test trigger it? :/

Also, fabbot refuses to run, there's this whitelist config to add authorized code to run. Any clue where to configure it? Never used it.

Reply:
> Says `testResolveIntersectionOfPaths()` invokes it, can't see even a generic call to it. Does the iterator test trigger it? :/

I think there are occurences in `tests/Fixtures/ConfigurationResolverPathsIntersection`.

> Also, fabbot refuses to run, there's this whitelist config to add authorized code to run. Any clue where to configure it? Never used it.

Maybe https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4828#discussion_r382710907 is related?